rolling-shutter/collator/batcher/batcher.go

```diff
@@ -232,6 +232,13 @@ func (btchr *Batcher) closeBatchImpl(ctx context.Context, db *cltrdb.Queries, l1
 		return err
 	}
 
+	// Lookup the current eon public key for the given block.
+	_, err = db.FindEonPublicKeyForBlock(ctx, l1blockNumber)
+	if err != nil {
```

Comment:
nitpick: this could also be a database failure or any other kind of error apart from `pgx.ErrNoRows`

Returning different errors for different conditions wouldn't change anything in the calling code though, since we don't catch specific errors there.

Reply:
Right, I think I am a bit lazy with the error handling in general.